Content.Server/Connection/ConnectionManager.cs

```diff
@@ -212,5 +409,24 @@ private async Task NetMgrOnConnecting(NetConnectingArgs e)
             await _db.AssignUserIdAsync(name, assigned);
             return assigned;
         }
+
+        private async Task<List<IAdminRemarksRecord>> GetAdminRemarks(NetUserId userId)
```

Comment:
You are never cleaning out this cache. It should be an object that is allocated in `IsWhitelisted` and not stored on the manager itself.